buildtest/cli/help.py

```diff
@@ -405,9 +405,20 @@ def print_history_help():
 
     table.add_row("buildtest history list", "List all build history files")
     table.add_row("buildtest history list --terse", "Print output in terse format")
+    table.add_row(
+        "buildtest history --pager list", "Paginate output of the history list"
```

Comment:
once change is done it should be `buildtest history list --pager` and not `buildtest history --pager list`